File: restaurant-Services/src/main/java/com/foody/restaurantServices/domain/Cuisine.java
package com.foody.restaurantServices.domain;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document
@AllArgsConstructor
@NoArgsConstructor
@Setter
@Getter
@ToString
public class Cuisine {

    @Id
    private String cuisineId;
    private String cuisineName;
    private String cuisineDescription;
    private int price;
    private int qty;
    private byte[] image;
    private int total;
}

File: src/app/model/Cuisine.ts
export class Cuisine{
    cuisineId:any;
    cuisineName:any;
    image:any;  
    price:any;
    qty:any;
    cuisineDescription:any;
    total:any


    constructor(
        cuisineId:any,
        cuisineName:any,
        image:any,
        price:any,
        qty:any,
        cuisineDescription:any,
        total :any

    ){
        this.cuisineId=cuisineId;
        this.cuisineName=cuisineName;
        this.image=image;
        this.price=price;
        this.qty=qty;
        this.cuisineDescription=cuisineDescription;
        this.total=total
        // this.rating=rating
    }

}